mitosheet/mitosheet/parser.py

```diff
@@ -767,6 +832,17 @@ def get_header_header_selection_code(column_header_one: ColumnHeader, column_hea
             transpiled_column_header = column_header_to_transpiled_code(column_header)
             replace_string = f'{df_name}[{transpiled_column_header}]{get_shift_string(column_dtype, row_offset)}' # type: ignore
         
+        elif match_type == '{SHEET}!{HEADER}:{HEADER}':
+            (sheet_name, column_header_one, column_header_two) = match['parsed']
+            cross_sheet_df = dfs[df_names.index(sheet_name)]
+            # Ensure that the column headers are in the order they appear in the dataframe
+            first_column_header, second_column_header = (column_header_one, column_header_two) if cross_sheet_df.columns.get_loc(column_header_one) <= cross_sheet_df.columns.get_loc(column_header_two) else (column_header_two, column_header_one) 
```

Comment:
If we don't have a test for this line of code (in `test_parser.py`) -- let's add one as well.